testdata/sqllogictests/explain.slt

```diff
@@ -5,3 +5,34 @@ explain select 1;
 
 statement ok
 explain analyze select 1;
+
+# Test for #1754
+# Ensure `RuntimeGroupExec` is pulled as far up as possible in the `EXPLAIN`ed output
+#
+# Omitting this test for now.
+# See https://github.com/GlareDB/glaredb/pull/1781#discussion_r1330072240
+
+halt
+
+statement ok
+create temp table t1754 as values (1, 'one'), (2, 'two'), (3, 'three');
+
+query TT
+explain
+select * from t1754
+where column1 != 2
+order by column2
+limit 1;
+----
+logical_plan Limit: skip=0, fetch=1
+  Sort: t1754.column2 ASC NULLS LAST, fetch=1
+    Filter: t1754.column1 != Int64(2)
+      TableScan: t1754 projection=[column1, column2]
+physical_plan RuntimeGroupExec: runtime_preference=local
+  GlobalLimitExec: skip=0, fetch=1
+    SortPreservingMergeExec: [column2@1 ASC NULLS LAST], fetch=1
+      SortExec: fetch=1, expr=[column2@1 ASC NULLS LAST]
+        CoalesceBatchesExec: target_batch_size=8192
+          FilterExec: column1@0 != 2
+            RepartitionExec: partitioning=RoundRobinBatch(10), input_partitions=1
```

Comment:
One concern here is that this may fail for people with different number of cores, check this run for instance (not sure how it passed here): https://github.com/GlareDB/glaredb/actions/runs/6235138391/job/16923733930

I'm ok with omitting this particular test from the pr, so that I can find some generic-enough variant at some later point.

Reply:
I'll just let this test be there. We can edit it later when you find a better alternative.

Reply:
Could we do something like what we do with replacing environment variables in queries? So we would have something like:

```
RepartitionExec: partitioning=RoundRobinBatch(${NUM_CPUS}), input_partitions=1
```

and we set `NUM_CPUS` in the test runner.

Reply:
Yeah, that sounds like a good solution.